bin/pocs_shell

```diff
@@ -193,9 +193,12 @@ Hardware names: {}   (or all for all hardware)'''.format(
             try:
                 self.pocs.run()
             except KeyboardInterrupt:
-                print_warning('POCS interrupted, skipping states and parking')
-                self.pocs.observatory.mount.home_and_park()
-                self._running = False
+                print_warning('POCS interrupted, parking')
```

Comment:
Perhaps add the TODO here for the behavior you suggested in the PR?

Reply:
Would prefer to move to #592 actually. Can make a note.